### PR TITLE
libtool: update to version 2.4.7

### PR DESCRIPTION
**Pull request by mmuetzel**
I built Octave on MINGW64 after upgrading to this version and it still seems to do its job.

### CODE REVIEW

File: libtool/PKGBUILD [hunk 1]
```diff
@@ -65,7 +76,7 @@ build() {
 
 check() {
   cd ${srcdir}/${pkgbase}-${pkgver}
-  make check
+  # make check
```
Comment (Biswa96) [inline libtool/PKGBUILD:79]: This commenting is not required. One can run check function in their system.

Comment (mmuetzel) [reply libtool/PKGBUILD:79]: The check sometimes got stuck for me for some reason. (But that already happened with 2.4.6 iirc.) So, I commented it out. I'll revert that.

Comment (Biswa96) [reply libtool/PKGBUILD:79]: You can write it something like this `make check || warning "Tests failed"`. Or you can disable check() while building a package with `makepkg-mingw --nocheck` command.

Comment (mmuetzel) [reply libtool/PKGBUILD:79]: The issue wasn't that the checks were failing. But they were stuck for a very long time (maybe indefinitely). Maybe using `timeout 15m make check || warning "Tests failed"` would also have helped...

Comment (Biswa96) [reply libtool/PKGBUILD:79]: Opinions may vary. Feel free to change that command as you like. Just I would suggest not to disable/comment out it.

Comment (mmuetzel) [reply libtool/PKGBUILD]: Done.

**Comment by mmuetzel** [comment]
I also have a patch that would allow linking to the Flang support libraries using libtool. It is kind of orthogonal to the version update.
Would it be ok to include it in this PR? Or better a separate PR?

**Comment by mmuetzel** [comment]
I figured I'll just add that patch for Flang in this PR. Please, let me know if that's not a good idea and I'll revert.

The idea is similar to patch `0013-Allow-statically-linking-compiler-support-libraries-.patch` by @mstorsjo: Add an exception for the support libraries provided by the compilers.
But since the Flang support libraries are added with `-l` flags instead of a full path to the static support library, the exception needs to be added slightly differently.

**Comment by lazka** [comment]
thanks!

**Pull request merged**